packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -92,6 +116,7 @@ export const PieChart: FC<PieChartProps> = ({
     return (
       <VizLegend
         items={legendItems}
+        eventBus={eventBus}
```

Comment:
Idea - panel event's bus available via context, so that we avoid props drilling through the viz components. WDYT?

Sth like:
```tsx
<PanelChrome>
  <EventsBusContext>
    <VizComponents />
  </EventsBusContext>
</PanelChrome>
```

Reply:
Using the context for passing eventBus looks interesting